tests/e2e/local/minikube/Troubleshooting.md

```diff
@@ -9,7 +9,7 @@ This file helps note down issues we have seen and how to debug them
    ```
 
    The `GOOS=linux` is required when you are running setup on macOS.
-1. If your docker push step failed, please check if you have port forwarding setup for localregistry or not and does docker know about insecure registry at localhost:5000 or not. You can run following script to do that as well.
+1. If your docker push step failed, please check if you have port forwarding setup for local registry or not and does docker know about insecure registry at localhost:5000 or not. You can run following script to do that as well.
```

Comment:
I think we can remove this since we do not use local register in the minikube case...

Reply:
Yes, I checked the codes and we can safely remove the docker push part in this file.

Reply:
Comments are fixed now, PTAL :)

Reply:
hmmmm, let me check the codes.